rebar.config

```diff
@@ -12,20 +12,17 @@
 		  {full , [{basic}, netlink, {amqp}, {collectd}]}]}.
 {deps,
  [
-  {exometer_core, ".*", {git, "git://github.com/Feuerlabs/exometer_core.git",
-			 {tag, "1.4"}}},
-  {exometer_collectd, ".*", {git, "git://github.com/Feuerlabs/exometer_collectd.git",
-			     {tag, "1.0.1"}}},
-  {netlink, ".*", {git, "git://github.com/Feuerlabs/netlink.git", "aab67b0"}},
-  {amqp_client, ".*", {git, "git://github.com/jbrisbin/amqp_client.git", {tag, "rabbitmq-3.3.5"}}},
-  {jiffy, ".*", {git, "git://github.com/davisp/jiffy.git", "0.13.3"}}
+  {exometer_core, ".*", {git, "git://github.com/Feuerlabs/exometer_core.git", "09c2a93"}},
+  {exometer_collectd, ".*", {git, "git://github.com/Feuerlabs/exometer_collectd.git", "tb/update_exometer_core"}},
+  {netlink, ".*", {git, "git://github.com/tolbrino/netlink.git", "tb/logging"}},
```

Comment:
This will be adapted in this PR, once the respective change has been merged: https://github.com/Feuerlabs/netlink/pull/8